client/src/app/site/motions/services/motion-pdf.service.ts

```diff
@@ -170,6 +191,56 @@ export class MotionPdfService {
         return motionPdfContent;
     }
 
+    private createAttachments(motion: ViewMotion): object {
+        const pageMarginLeft = this.pdfDocumentService.mmToPoints(
+            this.meetingSettingsService.instant(`export_pdf_page_margin_left`)
+        );
+        const pageMarginRight = this.pdfDocumentService.mmToPoints(
+            this.meetingSettingsService.instant(`export_pdf_page_margin_right`)
+        );
+        const pageSize = this.meetingSettingsService.instant(`export_pdf_pagesize`);
+        let width = pageSize === `A4` ? this.PDF_A4_POINTS_WIDTH : this.PDF_A5_POINTS_WIDTH;
+        width = width - pageMarginLeft - pageMarginRight;
+        const instancUrl = this.organizationSettingsService.instant(`url`);
+
+        const attachments = [];
+        attachments.push({
+            text: this.translate.instant(`Attachments`),
+            style: `heading3`,
+            margin: [0, 10, 0, 0]
+        });
```

Comment:
Should this always be appended or only when at least one attachment is exported?